src/ripple/app/paths/cursor/AdvanceNode.cpp

```diff
@@ -20,13 +20,18 @@
 #include <BeastConfig.h>
 #include <ripple/app/paths/cursor/RippleLiquidity.h>
 #include <ripple/basics/Log.h>
+#include <ripple/ledger/View.h>
 
 namespace ripple {
 namespace path {
 
 TER PathCursor::advanceNode (STAmount const& amount, bool reverse, bool callerHasLiquidity) const
 {
-    bool multi = multiQuality_ || (!callerHasLiquidity && amount == zero);
+    bool multi = multiQuality_;
+    if (dcSwitchover(view().info().parentCloseTime))
+        multi = multi || (!callerHasLiquidity && amount == zero);
```

Comment:
If we're going this route:

    bool const multi = dcSwitchover(view().info().parentCloseTime)
        ? (multiQuality_ || (!callerHasLiquidity && amount == zero))
        : (multiQuality_ || amount == zero);